lib/libc/gen/pwcache.c

```diff
@@ -2,9 +2,39 @@
 #include <grp.h>
 
 const char *user_from_uid(uid_t uid, int noname) {
```

Comment:
Since implementation of these is not based on `lib/libc/gen/pwcache.c` from _NetBSD_ it's worth to mention it here.

OTOH I guess it'd be possible to take [pwcache.c](http://bxr.su/NetBSD/lib/libc/gen/pwcache.c#260) as is and add required functions from [getpwent.c](http://bxr.su/NetBSD/distrib/utils/libhack/getpwent.c). Could you verify that?

Reply:
I will check that and push it in another PR.